### PR TITLE
add bridging info

### PR DESCRIPTION
**Pull request by vipineth**
[x] Add bridging info for tokens

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205276763751738

### CODE REVIEW

File: src/domain/synthetics/markets/getBridigingInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Trans } from "@lingui/macro";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import getBridgingOptionsForToken from "config/bridging";
+import { getChainName } from "config/chains";
+
+export default function getBridgingInfo(chainId: number, tokenSymbol?: string) {
```
Comment (divhead) [inline src/domain/synthetics/markets/getBridigingInfo.tsx:6]: not sure that this code relatred to markets. Also there is jsx and not some maths.  Probably better to create component BridgingInfo

Comment (vipineth) [reply src/domain/synthetics/markets/getBridigingInfo.tsx]: Thanks, for the feedback. Updated

File: src/config/bridging.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    },
+  ],
+};
+
+export function getBridgingOptionsForToken(tokenSymbol?: string): BridgingOptionsReturn {
+  if (!tokenSymbol) return;
+  return BRIDGING_OPTIONS[tokenSymbol];
+}
+
+export function isBridgingInfoAvailableForToken(tokenSymbol?: string): boolean {
```
Comment (divhead) [inline src/config/bridging.ts:31]: mm, probably we don't need this func as we could just use getBridgingOptionsForToken ?

File: src/config/bridging.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { ARBITRUM, AVALANCHE } from "./chains";
+
+const SOLANA_BRIDGING_IDS = {
+  [ARBITRUM]: 23,
+  [AVALANCHE]: 6,
+};
+
+type BridgingOptionsReturn =
```
Comment (divhead) [inline src/config/bridging.ts:8]: probaby just BridgingOptions?

File: src/config/bridging.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  | [
+      {
+        name: string;
+        generateLink: (chainId: number) => string;
+      }
+    ]
+  | undefined;
+
+const BRIDGING_OPTIONS = {
```
Comment (divhead) [inline src/config/bridging.ts:17]: would be good to specify type here to increase intellisense

BRIDGING_OPTIONS: {[symbol: string]: BridgingOptions[]}

**Review by divhead — APPROVED** [review]

**Pull request merged**